test/unit/specs/components/staking/__snapshots__/PageValidator.spec.js.snap

```diff
@@ -158,7 +158,7 @@ exports[`PageValidator has the expected html structure 1`] = `
                 class="address"
                 id="address"
               >
-                cosmoschiapudding…6789
+                …2b3c
```

Comment:
Test test uses some funky non-address value `1a2b3c`. I will change this later.

Reply:
ok, we should consider refactoring all the unit tests to use only the mocked values from `lcdClientMock` to avoid changing so many files when there's a breaking change renaming on the SDK